projects/ion/src/lib/defaultImage.directive.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/angular';
+import { DefaultImageDirective } from './defaultImage.directive';
+
+async function sut(src = 'test.png', defaultImage = 'default.png') {
```

Comment:
```suggestion
const defaultSrc = 'test.png';
async function sut(src = defaultSrc, defaultImage = 'default.png') {
```

Same to defaultImage.